file.go

```diff
@@ -159,12 +157,14 @@ func (f *win32File) prepareIo() (*ioOperation, error) {
 }
 
 // ioCompletionProcessor processes completed async IOs forever
-func ioCompletionProcessor(h syscall.Handle) {
+func ioCompletionProcessor(h windows.Handle) {
 	for {
 		var bytes uint32
-		var key uintptr
+		var key uint32
 		var op *ioOperation
-		err := getQueuedCompletionStatus(h, &bytes, &key, &op, syscall.INFINITE)
```

Comment:
It looks like `windows.GetQueuedCompletionStatus` taking a `*uint32` for `key` is a bug. The syscall definition takes a `PULONG_PTR`:

```
#if defined(_WIN64)
 typedef unsigned __int64 ULONG_PTR;
#else
 typedef unsigned long ULONG_PTR;
#endif

typedef ULONG_PTR *PULONG_PTR;
```

I think we will need to stick with the local `getQueuedCompletionStatus` definition until this is fixed in `golang.org/x/sys/windows`. @dcantah did we ever send a PR for this?

Reply:
Nope, I forgot about it as we just used a local binding in hcsshim also 😅. I'll send one today.

Reply:
https://github.com/golang/sys/commit/683adc9d29d7bd1f0f778e1f7f2eeb61b415f4d5#diff-6a5d6dfef9f877a293639aa3898a47ccf1f6be4ad6ce3e5e035d187f671a6e4b It was already fixed 😮

Reply:
@TBBle Might want to pull in the latest x/sys/window

Reply:
Done, thank you for catching that.